src/Raven.Client/Documents/DocumentStore.cs

```diff
@@ -405,10 +406,10 @@ public override CountersOperationExecutor Counters
             }
         }
 
-        public override BulkInsertOperation BulkInsert(string database = null)
+        public override BulkInsertOperation BulkInsert(string database = null, CancellationToken token = default)
```

Comment:
This is a breaking binary change, so only for 4.1, maintaining source compatability, right?

Reply:
Yes, I planned not do do a 4.0 PR at all for this issue.